### PR TITLE
Fix small style bugs and update features on homepage

### PR DESCRIPTION
**Pull request by kmcnellis**
Also add the logo pngs

### CODE REVIEW

File: client/app/main/main.html [hunk 1]
```diff
@@ -26,38 +22,41 @@
...
-      attending events. An instructor signs on and creates courses and events,
-      students can then use the venue app to upload event submissions, which
-      can contain images or GPS data that verifies a student attended an event.
+    <div class='row'>
+      <div class='col-md-6 icons'>
+        <span class='glyphicon glyphicon-phone'></span><span class='plus-text'> + </span>
+        <span class='glyphicon glyphicon-book'></span>
+      </div>
+      <div class='col-md-6'>
+        <h3>What is venue?</h3>
```
Comment (seveibar) [inline client/app/main/main.html:31]: Josh recommended we stay consistent with a capital V, I think I agree (logo stays lowercase tho) :penguin:

File: client/app/main/main.html [hunk 2]
```diff
@@ -26,38 +22,41 @@
...
+      <div class='col-md-6'>
+        <h3> Features </h3>
+        <ul>
+          <li>Create courses with multiple sections</li>
+          <li>Student and Instructor dashboards with activity feeds</li>
+          <li>Geo-tagged events and picture uploads</li>
+          <li>Email notifications</li>
+          <li>Companion iOS and Android apps</li>
+          <li>Administrator Control Panels</li>
+          <li>Bulk data downloads</li>
```
Comment (seveibar) [inline client/app/main/main.html:53]: ...bulk data downloads?

I think I liked the old line, maybe an IMO @leej42 second opinion?

File: client/app/main/main.less [hunk 3]
```diff
@@ -1,6 +1,8 @@
 .homepage{
+  h3{
+    font-family:Raleway;
```
Comment (seveibar) [inline client/app/main/main.less:3]: yea!

**Comment by seveibar** [comment]
I saw you imported the logos, but they're incorrectly named (`dark_venue` for the light logo- or am i confused?) and not used.

Also the bulk data downloads line i'd like to get a second opinion- i prefer describing student submission methods.

**Comment by seveibar** [comment]
(it's ok that you just imported the logos, I just wanted to make sure they were intentionally not used)

**Comment by seveibar** [comment]
Also I like our tag line and that was removed :/

**Comment by kmcnellis** [comment]
It was intentional - the venue_dark is for a dark background.  I didn't end up using them because they don't currently match our design.  And feel free to change the descriptions, they were only a suggestion.

**Comment by seveibar** [comment]
ahh okay sorry

**Pull request merged**